test/port_helpers.py

```diff
@@ -108,5 +111,5 @@ def bind_port(sock, host=HOST):
             sock.setsockopt(socket.SOL_SOCKET, socket.SO_EXCLUSIVEADDRUSE, 1)
 
     sock.bind((host, 0))
-    port = sock.getsockname()[1]
+    port: int = sock.getsockname()[1]
```

Comment:
I suggest adding an `assert isinstance(port, int)` instead of this annotation, then if for some reason the type *isn't* int, it will be caught instead of silently being incorrect.